app/configs/crashDataCard.tsx

```diff
@@ -1,13 +1,11 @@
 import { crashesColumns } from "./crashesColumns";
 
-
-/** Construct card-specific arrays of columns */
 export const crashDataCards = {
   summary: [
     crashesColumns.case_id,
     crashesColumns.crash_timestamp,
-    crashesColumns.fhe_collsn_id,
-    crashesColumns.rpt_city_id,
+    crashesColumns.collsn,
+    crashesColumns.city,
```

Comment:
before adding that `satisfies` magic to the column indexes (see my other comment), there was no type checking on these column names. You could have added a column to the summary section called `crashColumns.pizza` and typescript would not have blinked.